operations/research-and-development/product/release-process/release-overview.md

```diff
@@ -65,23 +63,29 @@ The goal is not to have the same release cadence for all products, but there may
    * Add clear Release Notes on PRs.
 5. **Achieve Customer Obsession by doing retrospectives on release issues and monitoring customer/community release bug reports after releases.**
    * This allows us to learn from issues so that they don’t happen again and to fix critical bugs asap.
-   * Retrospectives for issues and dot releases are important. Sample retrospective doc used for releases. Could also use Playbooks for this.
+   * Retrospectives for issues and dot releases are important.
    * Monitor community and customer reports in GitHub, Forum, Zendesk, and channels like Ask R&D, in partnership with the Support team.
 6. **Release new products tightly integrated with the Mattermost suite.**
    * E.g. Boards will ship as a plugin in June; Playbooks is a plugin. Plugins are the strategy for now, maybe there will be another strategy for the long-term.
 
+## Multi-product Release Processes
+ - The sample [Plugin Release Playbook](https://community.mattermost.com/playbooks/playbooks/f4oh16ardfbyfgkas1cb6intmw/preview) helps give an overview of needed steps for plugin releases.
+ - Please also refer to the [Playbooks release documentation](https://handbook.mattermost.com/operations/research-and-development/product/release-process/playbooks-release) and [Boards release documentation](https://handbook.mattermost.com/operations/research-and-development/product/release-process/boards-release).
+
 ## Overview of Release Cycles
 
-Currently Mattermost Cloud releases occur on a 2-week cycle, but the goal is to release more frequently. Cloud and self-managed PRs don’t need to be cherry-picked to releases, except for any last minute bug fixes that get merged after the release branch is cut, and any necessary hotfixes.
+Currently Mattermost Cloud releases occur on a biweekly cycle, but the goal is to release more frequently. Cloud and self-managed PRs may need to be cherry-picked to the release branches, such as regression bug fixes and hotfixes that get merged after the release branch has already been cut.
 
 Schedule for Mattermost Cloud releases:
 
-* \(T-7\): 7 working days prior to the release day, the `master` branch is merged to the Cloud branch.
-* \(T-0\): Release Day
+* \(T-6\): 7 working days prior to the release day, the `master` branch is merged to the Cloud branch and Cloud test servers are updated.
```

Comment:
wondering what is the reason to merge so early

Reply:
QA wants to have test servers updated by Friday morning (European time) latest, so that they have enough time to run automated tests and smoke tests.

Ibrahim completes the "fast-forward" on Wednesday or Thursday, and test servers are updated on Thursday or Friday morning.

Reply:
ok I see @amyblais thanks for the details